build.gradle

```diff
@@ -36,10 +36,13 @@ dependencies {
     compileOnly 'org.jenkins-ci.plugins:plain-credentials:1.0@jar'
 
     testCompile 'org.junit.jupiter:junit-jupiter-engine:5.6.2'
+    testCompile 'org.junit.jupiter:junit-jupiter-params:5.6.2'
     testCompile 'org.mockito:mockito-junit-jupiter:3.3.3'
+
+    testRuntime 'javax.servlet:javax.servlet-api:4.0.1'
```

Comment:
Unable to make the call to Jenkins.getInstanceOrNull() without this. We get the following error otherwise:

```
javax/servlet/ServletException
java.lang.NoClassDefFoundError: javax/servlet/ServletException
	at com.synopsys.integration.jenkins.JenkinsProxyHelper.getProxyInfoFromJenkins(JenkinsProxyHelper.java:44)
	at com.synopsys.integration.jenkins.JenkinsProxyHelperTest.testGetProxyInfoFromJenkins_NullJenkins(JenkinsProxyHelperTest.java:74)
```